config/networking/kustomization.yaml

```diff
@@ -0,0 +1,3 @@
+resources:
```

Comment:
Neither ingress nor route will be present in the release yaml and I think one of them (probably ingress, as its universal) should be there.